README.md

```diff
@@ -11,160 +8,142 @@
 |------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [![Install From AMO](https://ipfs.io/ipfs/QmWNa64XjA78QvK3zG2593bSMizkDXXcubDHjnRDYUivqt)<br>![](https://img.shields.io/amo/users/ipfs-companion?label=AMO%20users&style=social)](https://addons.mozilla.org/firefox/addon/ipfs-companion/) | [![Install from Chrome Store](https://ipfs.io/ipfs/QmU4Qm5YEKy5yHmdAgU2fD7PjZLgrYTUUbxTydqG2QK3TT)<br>![](https://img.shields.io/chrome-web-store/users/nibjojkomfdiaoajekhjakgkdhaomnch?label=Chrome%20Web%20Store%20users&style=social)](https://chrome.google.com/webstore/detail/ipfs-companion/nibjojkomfdiaoajekhjakgkdhaomnch) |
 
-## Lead Maintainer
+## Lead maintainer
 
-[Marcin Rataj](https://github.com/lidel)
+[Marcin Rataj](https://github.com/lidel), with help from the [IPFS GUI working group](https://github.com/ipfs/ipfs-gui).
 
-## Table of Contents
+## Table of contents
 
-- [Background](#background)
+- [About IPFS Companion](#about-ipfs-companion)
 - [Features](#features)
-- [Install](#install)
+- [Install](#install-ipfs-companion)
 - [Contribute](#contribute)
-- [Troubleshooting](#troubleshooting)
-- [Privacy Policy](#privacy-policy)
-- [License](#license)
+- [Help & Troubleshooting](#help--troubleshooting)
+- [Privacy & Licenses](#privacy--license)
 
-## Background
+## About IPFS Companion
 
-This add-on enables everyone to access IPFS resources the way they were meant: from locally running IPFS node :-)
+IPFS Companion harnesses the power of your locally running IPFS node (either through the IPFS Desktop app or the command-line daemon) directly inside your favorite Chromium-based or Firefox browser, enabling support for `ipfs://` addresses, automatic IPFS gateway loading of websites and file paths, easy IPFS file import and sharing, and more.
 
-IPFS is a new hypermedia distribution protocol, addressed by content and identities.
-IPFS enables the creation of completely distributed applications.
-It aims to make the web faster, safer, and more open.
+IPFS is a peer-to-peer hypermedia protocol designed to make the web faster, safer, more resilient, and more open. It enables the creation and dissemination of completely distributed sites and applications that don’t rely on centralized hosting and stay true to the original vision of an open, flat web. Visit [the IPFS Project website](https://ipfs.io) to learn more.
 
-Learn more at [ipfs.io](https://ipfs.io) (it is really cool, we promise!)
+## IPFS Companion features
```

Comment:
Link in ToC at the top is broken due to this change – unsure which one should be changed, up to you